internal/rbd/rbd_util.go

```diff
@@ -2273,3 +2221,28 @@ func (ri *rbdImage) GetClusterID(ctx context.Context) (string, error) {
 
 	return ri.ClusterID, nil
 }
+
+func (rv *rbdVolume) PrepareVolumeForSnapshot(ctx context.Context, cr *util.Credentials) error {
```

Comment:
no need to expose this function.

Reply:
> no need to expose this function.

Yup, we need to expose it in order to be able to call from Volume type struct variable.